web-client/src/ducks/requests/actions.ts

```diff
@@ -3,90 +3,40 @@ import { IRequest, Request } from 'src/models/requests';
 /* TODO: why rename? */
 import {
   createUserRequest,
-  getAcceptedRequest as getAcceptedRequestFunc,
-  getArchivedRequest as getArchivedRequestFunc,
-  getFinishedRequest as getFinishedRequestFunc,
-  getOngoingRequest as getOngoingRequestFunc,
-  getOpenRequest as getOpenRequestFunc,
-  observeOpenRequests as observeOpenRequestsFunc,
+  getAcceptedPost as getAcceptedPostFunc,
+  getOpenPost as getOpenPostFunc,
   setUserRequest,
 } from './functions';
 import {
   CHANGE_MODAL,
-  GET_ACCEPTED,
-  GET_ARCHIVED,
-  GET_FINISHED,
-  GET_ONGOING,
-  GET_OPEN,
-  IgetOpenRequests,
-  OBSERVE_OPEN_REQUESTS,
+  GET_OFFER_POST,
+  GET_REQUEST_POST,
+  IgetRequestPosts,
+  RESET_OFFER_POST,
+  RESET_REQUEST_POST,
   RESET_SET,
   SET,
   SET_TEMP_REQUEST,
 } from './types';
 
-export const getOpenRequests = (payload: IgetOpenRequests) => (
+export const getRequestPosts = (payload: IgetRequestPosts) => (
   dispatch: Function,
 ) =>
   dispatch({
-    type: GET_OPEN,
-    firebase: getOpenRequestFunc,
+    type: GET_REQUEST_POST,
+    firebase: getOpenPostFunc,
```

Comment:
This will only give you the posts that have status as open. We need posts of all statuses here. I have already mentioned as separate comment in the functions file for using firebase directly.

Reply:
I know, I want to do this as part of a separate pull request.  What is in develop branch has same issue.